init.rb

```diff
@@ -48,6 +48,18 @@
 
 current_flavor = Rock.flavors.current_flavor
 
+#The following option is used to determine which xml importer should be used.
+#Currently castxml is working
+
+configuration_option('TYPELIB_CXX_LOADER', 'string',
+    :default => 'castxml',
+    :possible_values => ['gccxml', 'castxml'],
+    :doc => [
+        "Which xml importer should be used?",
+        "Currently only castxml is working when using master flavor"])
```

Comment:
We are supposed to keep support for the old LTS until LTS+1 is out (i.e. 16.10)

Given the timeline, I would either (1) decide to drop that rule or (2) release master as-is RIGHT NOW and re-release with C++11 support at the end of the year.